TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
I think it boils down to using `--infer-runtimes` option in `dotnet restore` for restoring all our test assets. I see that if the project.json has a netstandard1.* TFM then by default the lock files do not have any RID targets. But when using `--infer-runtimes` the lock files do have a RID specific target.